src/main/java/frc/robot/Constants.java

```diff
@@ -7,6 +7,8 @@ public final class Constants {
   public static final int MANIPULATOR_PORT2 = 10;
   public static final double MANIPULATOR_POWER = 1.0;
```

Comment:
We can make it a bit more realistic
```suggestion
  public static final double MANIPULATOR_POWER = 0.8;
```